packages/styles/icon-button.css

```diff
@@ -152,61 +154,83 @@ a.IconButton {
 }
 
 .IconButton--error:active {
-  background-color: var(--icon-button-background-color-error-active);
+  background-color: var(--button-background-color-error-active);
 }
 
 /* Dark Theme */
 
 .cauldron--theme-dark {
-  --icon-button-border-color-secondary: var(--accent-medium);
-  --icon-button-background-color-secondary: var(--accent-medium);
-  --icon-button-background-color-primary: var(--gray-20);
   --icon-button-icon-color: var(--white);
-}
+  --icon-button-outline-shadow-color: var(--accent-medium);
 
-.cauldron--theme-dark .IconButton--error {
-  background-color: var(--accent-dark);
-  color: var(--white);
-}
+  /* Primary */
+  --icon-button-primary-color: var(--white);
+  --icon-button-background-color-primary: var(--accent-medium);
+  --icon-button-background-color-primary-active: var(--accent-info-active);
+  --icon-button-outline-color-primary: var(--accent-info);
 
-.cauldron--theme-dark .IconButton--primary {
-  color: var(--gray-90);
-}
+  /* Secondary */
+  --icon-button-border-color-secondary: var(--accent-light);
+  --icon-button-background-color-secondary: var(--accent-medium);
+  --icon-button-secondary-color: var(--white);
+  --icon-button-background-color-secondary-active: var(--accent-light);
+  --icon-button-outline-color-secondary: var(--accent-light);
 
-.cauldron--theme-dark .IconButton--primary[aria-disabled='true'],
-.cauldron--theme-dark .IconButton--primary[disabled] {
-  background-color: var(--gray-20);
-  color: var(--gray-40);
+  /* Error */
+  --icon-button-background-color-error: var(--accent-medium);
+  --icon-button-outline-color-error: var(--accent-danger);
 }
 
-.cauldron--theme-dark .IconButton--secondary[aria-disabled='true'],
-.cauldron--theme-dark .IconButton--secondary[disabled] {
-  color: #5d676f;
+.cauldron--theme-dark .IconButton--primary {
+  border: 2px solid var(--accent-info);
 }
 
 .cauldron--theme-dark .IconButton--secondary {
-  color: var(--white);
+  border-width: 2px;
 }
 
 .cauldron--theme-dark .IconButton--error {
-  background-color: var(--accent-danger);
-  color: var(--accent-dark);
+  border: 2px solid var(--accent-danger);
 }
 
-.cauldron--theme-dark .IconButton--primary:active {
-  background-color: #e5ebec;
+.cauldron--theme-dark .IconButton--primary[aria-disabled='true'],
+.cauldron--theme-dark .IconButton--primary[disabled],
+.cauldron--theme-dark .IconButton--secondary[aria-disabled='true'],
+.cauldron--theme-dark .IconButton--secondary[disabled],
+.cauldron--theme-dark .IconButton--error[aria-disabled='true'],
+.cauldron--theme-dark .IconButton--error[disabled] {
+  color: var(--dark-workspace-color);
```

Comment:
This should not referencing a global variable directly, but an aliased color. Something like `--icon-button-disabled-color`.